src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetInvitationsQueries/GetInvitations/GetInvitationsQueryHandlerTests.cs

```diff
@@ -824,6 +824,21 @@ public async Task Handler_ShouldReturnCorrectDto()
             }
         }
 
+        [TestMethod]
+        public async Task Handler_ShouldReturnMoreThanOneProjectName_When_RequestProjectName_Is_Null()
+        {
+            var filter = new Filter { };
+            using (var context =
+                new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetInvitationsQuery(null, null, filter);
+                var dut = new GetInvitationsQueryHandler(context, _permissionCache, _plantProvider, _currentUserProvider);
+
+                var result = await dut.Handle(query, default);
+                Assert.IsTrue(result.Data.Invitations.DistinctBy(x => x.ProjectName).Count() > 1);
```

Comment:
Also assert that permissionscache.GetProjectsForUserAsync has been called once

Reply:
Done